datafusion/sqllogictest/test_files/explain_tree.slt

```diff
@@ -280,7 +280,7 @@ physical_plan
 06)┌─────────────┴─────────────┐
 07)│       DataSourceExec      │
 08)│    --------------------   │
-09)│        bytes: 1072        │
+09)│        bytes: 1040        │
```

Comment:
I believe these files are now slightly smaller due to 

- https://github.com/apache/arrow-rs/pull/7555